requirements/base.txt

```diff
@@ -8,6 +8,8 @@ tabulate==0.8.6
 psycopg2-binary==2.8.4
 scipy==1.4.1
 overrides==3.1.0
+ansible==3.0.0
```

Comment:
is there any cheaper way to get the validation than ansible? That's a monster requirement for yaml validation